chstrings/__init__.py

```diff
@@ -21,6 +21,12 @@ def _preprocess_variables(config, strings):
     else:
         strings['lead_section_hint'] = ''
 
+    reliable_sources_hint_link = _link(
+        config.reliable_sources_link,
+        config.reliable_sources_link_title)
+    strings['reliable_sources_hint'] = \
+        flask.Markup(strings['reliable_sources_hint']) % reliable_sources_hint_link
```

Comment:
No problem, I just thought I'd leave some comments in case you were interested in following up with this, but I can also just do this myself later. We can leave the PR open as a reminder for myself (or if someone else wants to take it over), and I'll close it when I get to it.

For what it's worth, testing this should be easy (for someone who is comfortable with the command line and git): you'd just need to clone the repository, [install the dependencies](https://github.com/eggpi/citationhunt/blob/master/CONTRIBUTING.md#setting-up-a-local-citation-hunt) and run `./run-tests.sh`, no need to set up a database or anything. There would be some test failures because of the database, but the [import-checks](https://github.com/eggpi/citationhunt/blob/master/run-tests.sh#L8-L19) should be enough.

But anyway, I'll come back to this later if you don't, it already got me thinking about the implementation, which is nice.